docs/eventlisteners.md

```diff
@@ -194,3 +194,32 @@ spec:
       template:
         name: pipeline-template
 ```
+
+### CEL Interceptors
+
+CEL interceptors parse expressions to filter requests based on JSON bodies and request headers, using the
+[CEL](https://github.com/google/cel-go) expression language.
+
+Supported features include case-insensitive matching on request headers.
```

Comment:
Let's document the custom methods we add similar to https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions

Reply:
Is this ok for a start?

https://github.com/tektoncd/triggers/blob/5d0379587187bef6ced05c6469b10d272d28009f/docs/cel_expressions.md

Reply:
Looks good! :smile: Thanks!